packages/ember-engines/package.json

```diff
@@ -86,7 +86,7 @@
     "ember-addon"
   ],
   "dependencies": {
-    "@embroider/macros": "^0.43.5",
+    "@embroider/macros": "^0.50.2",
```

Comment:
@anehx `@embroider/macros` is now `1.0.0` so we should be able to do just `"@embroider/macros": "^1.0.0"`

Reply:
Done!